monitor/flowgen/util/util.go

```diff
@@ -26,3 +35,51 @@ func dec(amt float64, decimals int) *big.Int {
 
 	return new(big.Int).SetUint64(uint64(p))
 }
+
+// ApproveToken gives `user` max allowance for `token`.
+func ApproveToken(ctx context.Context, backend *ethbackend.Backend, token, user, contract common.Address) error {
```

Comment:
this is duplicated from `solver/app`, at least add a TODO to remove duplication.

Also, I think the comment can be improved: `ApproveToken gives the 'contract' max allowance to spend the 'user's 'tokens'.`